src/index.html

```diff
@@ -7,12 +7,60 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image">
+        <img
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+        />
+      </div>
+
+      <div class="card__content">
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="stars card__stars">
+            <div class="stars">
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star"></div>
+            </div>
+          </div>
+          <p class="card__quantity-reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          Price:
+          <span class="card__price card__price--bold">$2,199</span>
```

Comment:
The element with class 'card__price' is used twice, once as a block and once as an element. According to BEM methodology, a class should either be a block or an element, not both. Consider renaming the 'span' element to 'card__price-value' or something similar to avoid confusion and maintain BEM conventions.